packages/foreman/nodejs-datatables.net/nodejs-datatables.net.spec

```diff
@@ -0,0 +1,47 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name datatables.net
+
+Name: %{?scl_prefix}nodejs-datatables.net
```

Comment:
Once this is in, we can unbundle `nodejs-datatables.net-bs` as that only has `nodejs-datatables.net` and `nodejs-jquery` as deps, and those are both packaged then.

Reply:
Yes, that's in https://github.com/theforeman/foreman-packaging/pull/12174 already